tests/Feature/FileAdder/MediaConversions/MediaCollectionTest.php

```diff
@@ -170,9 +170,32 @@ public function registerMediaCollections()
 
         $model = $testModel::create(['name' => 'testmodel']);
 
+        $model->addMedia($this->getTestJpg())->preservingOriginal()->toMediaCollection('images');
         $model->addMedia($this->getTestJpg())->preservingOriginal()->toMediaCollection('images');
         $model->addMedia($this->getTestJpg())->preservingOriginal()->toMediaCollection('images');
 
         $this->assertCount(1, $model->getMedia('images'));
     }
+
+    /** @test */
+    public function if_the_only_keeps_latest_method_is_specified_it_will_delete_all_other_media_and_will_only_keep_the_latest_n_ones()
```

Comment:
Could this test get extended by any custom property which clearly identifies the four images to check that the latest 3 are still there? 
The current test only checks the count - which was okay for the single one, because all got deleted except the added one. But now we want to really just delete the first one.

Reply:
Added an assertion which checks wether the first file is no longer part of the returned collection of Media.

Reply:
https://github.com/spatie/laravel-medialibrary/pull/1533/commits/3fdf17ee03d635c396c93bc37067209f4cce2bfd